services/QuillLMS/lib/tasks/add_grammar_api_rules_and_feedback.rake

```diff
@@ -0,0 +1,31 @@
+namespace :grammar_api_rules_and_feedback do
+    desc 'data migration for grammar-api specific rules and feedback'
+    task :insert => :environment do
+        rules_csv = CSV.parse(File.read('lib/data/rules.csv'), headers: true)
+        valid_grammar_rules = rules_csv.select do |r| 
+            r['Module'] == 'Grammar API' && r['Rule UID'].present? 
+        end
+
+        ActiveRecord::Base.transaction do 
+            valid_grammar_rules.each do |r|
+                created_rule = Comprehension::Rule.create!(
+                    uid: r['Rule UID'],
+                    name: r['Rule'],
+                    description: r['Rule Description'],
+                    universal: true,
+                    rule_type: 'grammar',
+                    optimal: false,
+                    suborder: 1,
```

Comment:
This isn't super critical, but I think we actually want to put something here if we have it.  It's not dead-on, but I suggest using `Overall Rule Priority`.  Alternatively, since it looks like the CSV is in priority order, maybe we could just use the `index + 1` of where we are in the `each` call.